syntax = "proto3";

import "file-system-commands.proto";
import "qroma-comm-config-commands.proto";



message QromaCommCommand {
  oneof command {
    bytes appCommand = 1;
    FileSystemCommand fsCommand = 2;
    QromaCommConfigCommand commConfigCommand = 3;
  }
}

message QromaCommResponse {
  oneof response {
    bytes appResponse = 1;
    FileSystemResponse fsResponse = 2;
    QromaCommConfigCommand commConfigResponse = 3;
  }
}

